packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -205,7 +205,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
                 onInputValueChange={ this.onInputChange }
                 suggestedDisplayValue={ suggestedDisplayValue }
                 aria-activedescendant={ activeDescendant }
-                aria-owns='suggestion-list'
+                aria-owns={ !!this.state.suggestionsVisible ? 'suggestion-list' : '' }
```

Comment:
>this.state.suggestionsVisible [](start = 30, length = 29)

I don;t think you really need to use the !! prefix

Reply:
True, because it's a ternary. Removing.